### PR TITLE
Lesson3

### PR DESCRIPTION
**Pull request by JuliaBal**
Created a new branch, which based on master branch.  Completed the first task.

### CODE REVIEW

File: Lesson3_Ex2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    {
+        public abstract int AreaCalculator();
+        public abstract int PerimeterCalculator();
+    }
+    
+    class Program
+    {
+        static void Main()
+        {
+
```
Comment (NaGriS) [inline Lesson3_Ex2/Program.cs:69]: ??? if there are any questions we could discuss it :)

File: Lesson3_Ex1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+namespace Lesson3_Ex1
+{
+    class Program
+    {
+        public static void Main()
+        {
+            Console.WriteLine("Please enter size of matrix: ");
+            string strsize = Console.ReadLine();
+            bool isInteger = int.TryParse(strsize, out int size);
```
Comment (NaGriS) [inline Lesson3_Ex1/Program.cs]: Could be moved to a separate method, as in your previous homeworks

**Review by NaGriS — COMMENTED** [review]
Waiting for the Figures task, first app is good

File: Lesson3_Ex2/Program.cs (hunk 1 above)
Comment (JuliaBal) [reply Lesson3_Ex2/Program.cs:69]: too many words in the task....

**Pull request closed without merging**